av1/av1codecconfigurationrecord.go

```diff
@@ -0,0 +1,99 @@
+package av1
+
+import (
+	"io"
+
+	"github.com/Eyevinn/mp4ff/bits"
+)
+
+// DecConfRec - AV1CodecConfigurationRecord
+// Specified in https://github.com/AOMediaCodec/av1-isobmff/releases/tag/v1.2.0
+type DecConfRec struct {
+	Marker                           byte
+	Version                          byte
```

Comment:
Only Version 1 is defined in the document, so I think we should return an error if the version is not 1. I still think it is fine to store the value.